com.unity.render-pipelines.core/Editor/Camera/CameraUI.Drawers.cs

```diff
@@ -0,0 +1,166 @@
+using System.Linq;
+using UnityEngine;
+
+namespace UnityEditor.Rendering
+{
+    /// <summary> Camera UI Shared Properties among SRP</summary>
+    public static partial class CameraUI
+    {
+        /// <summary>Enum to store know the expanded state of a expandable section on the camera inspector</summary>
+        public enum Expandable
+        {
+            /// <summary> Projection</summary>
+            Projection = 1 << 0,
+            /// <summary> Physical</summary>
+            Physical = 1 << 1,
+            /// <summary> Output</summary>
+            Output = 1 << 2,
+            /// <summary> Orthographic</summary>
+            Orthographic = 1 << 3,
+            /// <summary> RenderLoop</summary>
+            RenderLoop = 1 << 4,
+            /// <summary> Rendering</summary>
+            Rendering = 1 << 5,
+            /// <summary> Environment</summary>
+            Environment = 1 << 6,
+        }
```

Comment:
Just a small concern here now that we share the Expendable: 
- does every SRP will always have the same Area?
- Though as registration of expendable state is done in HDRP camera, the collapsing of lets say Projection area only collapse Projection for HDRP Camera and not for all. Is this what we want? Should we also have a shared expendable state? What is expected to be done when making a custom SRP now regarding this change ?

Reply:
URP will have an extra expandable state here that will be `Stack`,  I will add it on the future. The expanded state isn't shared. Maybe we can share it, it will be an option.

A custom SRP could have it's own store for it expandables. The thing here is we are letting SRP to use this sections, if they want to use it is on their hands.